lib/importproject.cpp

```diff
@@ -167,18 +167,20 @@ void ImportProject::FileSettings::setIncludePaths(const std::string &basepath, c
 void ImportProject::import(const std::string &filename)
 {
     std::ifstream fin(filename);
+
+	std::string path(Path::getPathFromFilename(Path::fromNativeSeparators(filename)));
+	if (!path.empty() && !endsWith(path, '/'))
+		path += '/';
+
     if (!fin.is_open())
         return;
     if (filename.find("compile_commands.json") != std::string::npos) {
         importCompileCommands(fin);
     } else if (filename.find(".sln") != std::string::npos) {
-        std::string path(Path::getPathFromFilename(Path::fromNativeSeparators(filename)));
-        if (!path.empty() && !endsWith(path,'/'))
-            path += '/';
-        importSln(fin,path);
+        importSln(fin, path);
     } else if (filename.find(".vcxproj") != std::string::npos) {
         std::map<std::string, std::string> variables;
-        importVcxproj(filename, variables, emptyString);
+        importVcxproj(filename, path, variables, emptyString);
```

Comment:
Yes, exactly, the problem exists when the solution contains absolute paths to the projects